app/src/main/java/leo/me/la/codeblue/ar/AugmentedImageNode.kt

```diff
@@ -0,0 +1,57 @@
+package leo.me.la.codeblue.ar
+
+import android.content.Context
+import android.net.Uri
+import android.os.Build
+import android.support.annotation.RequiresApi
+import android.util.Log
+import com.google.ar.core.AugmentedImage
+import com.google.ar.core.Pose
+import com.google.ar.sceneform.AnchorNode
+import com.google.ar.sceneform.Node
+import com.google.ar.sceneform.math.Quaternion
+import com.google.ar.sceneform.math.Vector3
+import com.google.ar.sceneform.rendering.ModelRenderable
+import java.util.concurrent.CompletableFuture
+
+class AugmentedImageNode : AnchorNode() {
```

Comment:
We probably don't need this. But let remove it later when we have pair programming session. For now, it is OK to put it here